app/javascript/packages/components/troubleshooting-options.jsx

```diff
@@ -11,17 +11,20 @@ import { BlockLink } from '@18f/identity-components';
 /**
  * @typedef TroubleshootingOptionsProps
  *
- * @prop {string} heading
+ * @prop {string=} headingTag
+ * @prop {string} headingText
  * @prop {TroubleshootingOption[]} options
  */
 
 /**
  * @param {TroubleshootingOptionsProps} props
  */
-function TroubleshootingOptions({ heading, options }) {
+function TroubleshootingOptions({ headingTag = 'h2', headingText, options }) {
```

Comment:
Could we call it `headingLevel` to match the ERB partial? Or vice-versa, update the ERB partial to match this name?

I suppose an argument could be made that "heading level" would be the numeric value, not the tag name, so either renaming both as "heading tag", or using "heading level" as a numeric value.